src/components/board/Stack.vue

```diff
@@ -40,6 +40,9 @@
 				</form>
 			</transition>
 			<Actions v-if="canManage" :force-menu="true">
+				<ActionButton icon="icon-clone" @click="cloneStack(stack)">
```

Comment:
Depending if we switch to write permissions, this needs to be adjusted as well

Reply:
ok, what would you like to do here?

Reply:
So cloning a stack is an operation that everyone with write permissions should be able to do, so this is not the proper place since it is only shown with canManage.